request.md

```diff
@@ -34,16 +34,22 @@
   </tr>
 </table>
 
+6) Please provide a link to the documentation for this data collection which describes the ultimate data set in a public, complete, and accurate way.
+ * Often the Privacy Notice for your product will link to where the documentation is expected to be.
+ * Common examples for Mozilla products/services:
+    * If this collection is Telemetry you can state "This collection is documented in its definitions files Histograms.json, Scalars.yaml, and/or Events.yaml and in the Probe Dictionary at https://probes.telemetry.mozilla.org."
```

Comment:
should we give links to the various json/yaml files here, or do we feel that the probe dictionary link is sufficient?

Reply:
The definitions files aren't really documentation in that they don't 100% meet the "human readable" "standard" we're aiming for in the way that searching on probe dictionary does. As such I think we can get away without highlighting them too much. (one of the reasons we include them at all is to highlight that if it's not Telemetry that's in those files (like if it's a custom ping or a part of the Environment) you can't use this example text. Otherwise we could drop them.)